src/LanguageServer/Test/FluentAssertions/DocstringConverterAssertions.cs

```diff
@@ -23,12 +23,12 @@ namespace Microsoft.Python.LanguageServer.Tests.FluentAssertions {
     [ExcludeFromCodeCoverage]
     internal static class DocstringAssertions {
         public static AndConstraint<StringAssertions> ConvertToPlaintext(this StringAssertions a, string plaintext, string because = "", params object[] reasonArgs) {
-            DocstringConverter.ToPlaintext(a.Subject.NormalizeLineEndings()).Should().Be(plaintext.NormalizeLineEndings().TrimEnd(), because, reasonArgs);
+            DocstringConverter.ToPlaintext(a.Subject?.NormalizeLineEndings()).Should().Be(plaintext?.NormalizeLineEndings().TrimEnd(), because, reasonArgs);
```

Comment:
Since this is the only place `NormalizeLineEndings` is used, we can check for `null` in the extension method itself.

Reply:
`plaintext` is an expected string, we shouldn't change it in assertion.

Reply:
something like `DocstringConverter.ToPlaintext(a.Subject.NormalizeLineEndings()).Should().Be(plaintext, because, reasonArgs);`

Reply:
If I don't normalize the expected output, then all of my other tests will need to make this call, as they would depend on the line endings of the source file and the system line endings.

Reply:
Hm... ok, let's leave it as it is.